pkg/common/redis.go

```diff
@@ -238,6 +239,11 @@ func (r *RedisClient) handleChannelSubs(
 				return
 			}
 
+			if message == nil {
+				errCh <- ErrNilMessage
+				return
+			}
```

Comment:
I saw a worker crash with a nil pointer error in the EventBus' receive function. It was trying to consume a message that was `nil`.